logfire/_internal/config_params.py

```diff
@@ -98,7 +98,9 @@ class _DefaultCallback:
 """Whether to enable the f-string magic feature. On by default for Python 3.11 and above."""
 IGNORE_NO_CONFIG = ConfigParam(env_vars=['LOGFIRE_IGNORE_NO_CONFIG'], allow_file_config=True, default=False, tp=bool)
 """Whether to show a warning message if logfire if used without calling logfire.configure()"""
-BASE_URL = ConfigParam(env_vars=['LOGFIRE_BASE_URL'], allow_file_config=True, default=LOGFIRE_BASE_URL)
+BASE_URL = ConfigParam(env_vars=['LOGFIRE_BASE_URL'], allow_file_config=True, default=None, tp=Union[str, None])
+REGION = ConfigParam(env_vars=['LOGFIRE_REGION'], allow_file_config=True, default='us')
```

Comment:
Maybe we also want the default to be `None` here?

I guess when a user logs in with the CLI or creates a new account prompted by the CLI they need to tell the CLI which region they want to log into, then the CLI opens the appropriate site.

Reply:
Yes I think we will need something like that. I'm not sure what the solution is, but if users have already signed up for an account and then get sent to a site where they don't have an account, that's a bad UX.

But I think we should be able to merge something that doesn't need this `REGION` param and purely helps with the case where a token is specified.

Reply:
If the token is specified / we have their credentials we should be able to derive the region from that (platform PR we worked on together).